### ISSUE TITLE
Log filename chage from dotenv.

### ISSUE THREAD

**Issue by parkmira** [cakephp/app#618]
https://github.com/cakephp/app/blob/2734ac7bdf9e69d5053faf14096c4e5c46260420/config/.env.default#L34

I would like to have `debug.log` and `error.log` under the logs. 
But `logsdebug.log`, `logserror.log`

```
#export LOG_DEBUG_URL="file://logs?levels[]=notice&levels[]=info&levels[]=debug&file=debug"
#export LOG_ERROR_URL="file://logs?levels[]=warning&levels[]=error&levels[]=critical&levels[]=alert&levels[]=emergency&file=error"
```
↓I want change.
```
#export LOG_DEBUG_URL="file://logs/?levels[]=notice&levels[]=info&levels[]=debug&file=debug"
#export LOG_ERROR_URL="file://logs/?levels[]=warning&levels[]=error&levels[]=critical&levels[]=alert&levels[]=emergency&file=error"
```

**Comment by parkmira** [comment]
I will do it.

**Comment by markstory** [comment]
Fixed in #619
